packages/Tabs/src/Tabs.js

```diff
@@ -26,73 +25,95 @@ export default function Tabs(props) {
     tabHeight,
   } = props;
 
-  const indexToUse = index !== null ? index : defaultIndex;
-  const [activeIndex, setActiveIndex] = React.useState(indexToUse);
-  const [currentFocusIndex, setFocusIndex] = React.useState(indexToUse);
-  let tabListRef = React.useRef(null);
+  const [idTabs] = React.useState(uuidv4());
+  const refList = React.useRef(null);
 
-  React.useEffect(() => {
-    setActiveIndex(indexToUse);
-  }, [indexToUse, setActiveIndex]);
+  const isControlled = index !== undefined;
+  const initIndex = defaultIndex === undefined ? 0 : defaultIndex;
```

Comment:
This should mimic the previous behaviour with the default value for `defaultIndex` of `0`.  This is why I'm not considering this a breaking change.